src/common/utils/snapshots.ts

```diff
@@ -25,3 +26,26 @@ export function snapshotUrl(snapshotNum: string | number) {
 export function currentSnapshot(): number {
   return snapshotFromUrl(SNAPSHOT_URL);
 }
+
+/** Checks for ?snapshot=xyz in the querystring and returns it as a number. */
+export function getSnapshotOverride(): number | null {
+  if (typeof window !== 'undefined') {
+    const search = window?.location?.search;
+    if (search !== undefined) {
+      const params = QueryString.parse(search);
+      if ('snapshot' in params) {
+        const snapshot = Number(params['snapshot']);
+        if (snapshot !== Number.NaN && snapshot !== 0) {
+          return snapshot;
+        }
+      }
```

Comment:
I think you can use `QueryString.parse(search, { parseNumbers: true })` to get the parameter already parsed.

Reply:
Ah, cool.  Done!